### PR TITLE
added author to available variables, updated readme and fixed bug

### PR DESCRIPTION
**Pull request by jingniwei**
In compound-ad-client we want to be able to add an author to the path of the file uploaded. This PR adds this as an option. https://github.com/Tapjoy/compound-ad-client/pull/209

@deanxorr @Tapjoy/dmctv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ s3d .s3.production
 s3d .s3.staging
 s3d .s3.local
 ```
+# pass in custom author name
```
Comment (elaineuy) [inline README.md]: i would add a note in the read me that the author name is used to name the nested folder for the uploaded files

**Comment by elaineuy** [comment]
noob question, is there a way or a need for these scripts to have tests?

**Comment by trflagg** [comment]
@elaineuy There are tests, and I think this can be tested.  Check out https://github.com/Tapjoy/s3d/blob/master/test/shared_test.rb#L65

**Comment by deanxorr** [comment]
:+1: for the tests comment, though I think I'm mostly here since we paired on this. Any dumb logic things are probably my fault 😄

**Comment by trflagg** [comment]
Just spoke to @ehealy about testing to make sure this doesn't break the dashboard-react-client deploys.

We can build a custom slug of this from the branch and deploy it to a QA version of deployboard. We should do this once the code is finalized but before we merge

**Comment by jingniwei** [comment]
@ehealy and I tested this on the QA deploy board and have verified this is working. I am merging this into master.

**Pull request merged**